src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -148,6 +148,24 @@ export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanProperti
 		this.populateTable(columns, tableRows);
 	}
 
+	public expandOrCollapsePropertiesTable(expand: boolean): void {
```

Comment:
I'd suggest naming this `setPropertyRowsExpanded` - that's a bit clearer to me about what exactly it's doing (and less verbose). 

Also add a js doc that mentions it does this recursively too